share/completions/feh.fish

```diff
@@ -0,0 +1,89 @@
+complete -c feh -l action -s A -x -d "Specify a shell command as action to perform on the image"
+
+for i in (seq 1 9)
+	complete -c feh -l action$i -x -d "Extra action triggered with number key "$i
+end
+
+complete -c feh -l auto-zoom -s Z -d "Zoom pictures to screen size"
+complete -c feh -l borderless -s x -d "Create borderless window"
+complete -c feh -l cache-thumbnails -s P -d "Enable thumbnail caching"
+complete -c feh -l caption-path -s K -r -d "Path to directory containing image caption"
+complete -c feh -l customlist -s L -x -d "Print image info according to given format"
+complete -c feh -l cycle-once -d "Exit feh after one loop through the slideshow"
+complete -c feh -l draw-actions -s G -d "Draw the defined actions"
+complete -c feh -l draw-exif -d "Display some EXIF information"
+complete -c feh -l draw-filename -s d -d "Draw filename"
+complete -c feh -l draw-tinted -d "Show overlay texts on semi-transparent background"
+complete -c feh -l filelist -s f -r
+complete -c feh -l font -s e -x -r -d "Set global font"
+complete -c feh -l fontpath -s C -r -d "Add given path as directory to search fonts"
+complete -c feh -l force-aliasing -d "Disable antialiasing for zooming, bg settings, etc."
+complete -c feh -l fullindex -s I
+complete -c feh -l fullscreen -s F -d "Make window fullscreen"
+complete -c feh -l geometry -s g -x -d "Limit window size. Format: [w x h] [+ x + y]"
```

Comment:
Not knowing feh, I don't get that format. Is it something like "1920 x 1080 + 7 + 4"? What does the "+" stand for?